tools/get_weights_caffe.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+
+import argparse
+import h5py
+import json
+import logging
+import numpy as np
+import os
+import sys
+
+# Add path for DIGITS package
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+import digits.config
+digits.config.load_config()
+
+from digits import utils, log
+from digits.inference.errors import InferenceError
+
+# must call digits.config.load_config() before caffe to set the path
+import caffe
+
+logger = logging.getLogger('digits.tools.inference')
+
+"""
+Get weights from a pretrained model
+"""
+
+def get_weights(output_dir,net):
+
+    f = h5py.File(output_dir+'/filters.hdf5','a')
+
+    # load the image and perform a forward pass:
+    try:
+        net.forward()
```

Comment:
Why do you need to call `forward()`? Can't you access the weights without a forward pass?